Libraries/IntelliSenseXml/IntelliSenseXmlCommentsContainer.cs

```diff
@@ -85,12 +86,15 @@ private IEnumerable<FileInfo> EnumerateFiles()
 
         private void LoadFile(FileInfo fileInfo, bool printSuccess)
         {
-            if (!fileInfo.Exists)
+            try
             {
-                throw new Exception($"The IntelliSense xml file does not exist: {fileInfo.FullName}");
+                xDoc = XDocument.Load(fileInfo.FullName);
+            }
+            catch (Exception ex)
+            {
+                Log.Error(string.Format("Failed to load '{0}'. {1}", fileInfo.FullName, ex.ToString()));
```

Comment:
Suggestion:

```suggestion
                Log.Error($"Failed to load '{fileInfo.FullName}'. {ex}");
```

Reply:
Ha, sometimes my C# regresses to circa 2005.